src/Button/RedliningButton/RedliningButton.less

```diff
@@ -0,0 +1,26 @@
+@import '../../style/variables.less';
+
+button.react-geo-redliningbutton {
+  color: #fff;
+  background-color: @primary-color;
+  border-color: @primary-color;
+
+  &:focus{
+    background-color: lighten(@primary-color, 10);
+    border-color: lighten(@component-background, 10);
+  }
+
+  &:hover {
+    background-color: lighten(@primary-color, 10);
+    border-color: lighten(@component-background, 10);
+  }
+
+  &:disabled {
+    background-color: @component-background;
+    border-color: darken(@component-background, 10);
+  }
+}
+
+.wrapper {
```

Comment:
If the wrapper can't be avoided, please rename to `react-geo-redlining-button-wrapper` or similar.

Reply:
done in 08a4ce5